src/webhooks/registry.ts

```diff
@@ -19,12 +19,26 @@ import {
 } from './types';
 
 interface RegistryInterface {
-  webhookRegistry: WebhookRegistryEntry[];
+  webhookRegistry: {[topic: string]: WebhookRegistryEntry;};
+
+  /**
+   * Sets the handler for the given topic. If a handler was previously set for the same topic, it will be overridden.
+   *
+   * @param options Paramters to add a handler which are path, topic and webHookHandler
+   */
+  addHandler(options: WebhookRegistryEntry): Promise<void>;
+
+  /**
+   * Fetches the handler for the given topic. Returns null if no handler was registered.
+   *
+   * @param topic The topic to check
+   */
+  getHandler(topic: string): Promise<WebhookRegistryEntry | null>;
```

Comment:
I'm pretty sure these methods don't need to return Promises, since they're just doing dictionary operations which don't need to be deferred.

Reply:
ohhh this would make my life much easier